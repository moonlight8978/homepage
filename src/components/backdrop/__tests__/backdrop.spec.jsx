import React from 'react'
import { render } from '@testing-library/react'

import Backdrop from '../backdrop'

test('renders correctly', () => {
  const children = <p>Hello</p>
  const { container } = render(<Backdrop>{children}</Backdrop>)
  expect(container.firstChild).toMatchSnapshot()
})

test('accepts overiding className', () => {
  const children = <p>Hello</p>
  const { container } = render(<Backdrop className="abc">{children}</Backdrop>)
  expect(container.firstChild).toMatchSnapshot()
})
