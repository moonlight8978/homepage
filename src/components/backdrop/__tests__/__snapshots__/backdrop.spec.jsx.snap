// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`accepts overiding className 1`] = `
<div
  class="backdrop abc"
>
  <p>
    Hello
  </p>
</div>
`;

exports[`renders correctly 1`] = `
<div
  class="backdrop"
>
  <p>
    Hello
  </p>
</div>
`;
